arangod/VocBase/Methods/Collections.cpp

```diff
@@ -226,22 +227,21 @@ void Collections::enumerate(TRI_vocbase_t* vocbase,
   }
   std::vector<CollectionCreationInfo> infos{{name, collectionType, properties}};
   std::vector<std::shared_ptr<LogicalCollection>> collections;
-  Result res = create(vocbase, infos, createWaitsForSyncReplication,
-                      enforceReplicationFactor, isNewDatabase, nullptr,
-                      collections);
+  Result res = create(vocbase, options, infos, createWaitsForSyncReplication,
+                      enforceReplicationFactor, isNewDatabase, nullptr, collections);
   if (res.ok() && collections.size() > 0) {
     ret = std::move(collections[0]);
   }
   return res;
 }
 
-Result Collections::create(TRI_vocbase_t& vocbase,
+Result Collections::create(TRI_vocbase_t& vocbase, OperationOptions const& options,
                            std::vector<CollectionCreationInfo> const& infos,
                            bool createWaitsForSyncReplication,
                            bool enforceReplicationFactor, bool isNewDatabase,
                            std::shared_ptr<LogicalCollection> const& colToDistributeShardsLike,
                            std::vector<std::shared_ptr<LogicalCollection>>& ret) {
-  ExecContext const& exec = ExecContext::current();
```

Comment:
but isn't this the same thing still ? It is a bit confusing to use the OperationOptions if the only thing used is the ExecContext